src/Microsoft.Framework.TestHost/ApplicationHost.cs

```diff
@@ -0,0 +1,251 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Framework.TestHost.Commandline;
+using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Common;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace Microsoft.Framework.ApplicationHost
+{
+    public class Program
+    {
+        private readonly IAssemblyLoaderContainer _container;
+        private readonly IApplicationEnvironment _environment;
+        private readonly IServiceProvider _serviceProvider;
+
+        public Program(IAssemblyLoaderContainer container, IApplicationEnvironment environment, IServiceProvider serviceProvider)
```

Comment:
Ignore this. I started with a copy of the `ApplicationHost.Program` and worked backwards. Left it in for comparison purposes.